### ISSUE TITLE
[BUG] AdyenSwiftUI crash: UIViewControllerRepresentables must be value types: FullScreenView

### ISSUE THREAD

**Issue by xzoky** [Adyen/adyen-ios#1026]
**Describe the bug**
Attempting to use AdyenSwiftUI's `present(viewController:)` causes a crash on iOS 16.

**To Reproduce**
1. Add `.present(viewController:)` to a SwiftUI view.
2. Present the view

**Expected behavior**
I expect the view to load normally

**Smartphone (please complete the following information):**
 - Device: iOS Simulator
 - OS: iOS 16.0
 - SDK Version: 4.10.0

**Additional context**
This message gets printed to the console:
```
SwiftUI/UIViewControllerRepresentable.swift:332: Fatal error: UIViewControllerRepresentables must be value types: FullScreenView
```

Looks like the issue was fixed in https://github.com/Adyen/adyen-ios/pull/1007, but for some reason it's not part of [4.10.0](https://github.com/Adyen/adyen-ios/releases/tag/4.10.0).

**Comment by descorp** [comment]
Hey @xzoky 

Indeed, we only added it to v5 on `develop` branch 🤔
Sorry for the inconvenience - we will patch v4 as well

In a meanwhile you can just copy `FullScreenView` locally with the right type.

Thanks for feedback! 💚

**Comment by mohammedDehairy** [comment]
@xzoky , can you check [this pull request](https://github.com/Adyen/adyen-ios/pull/1034) and confirm that its fixes the issue, we'll release this fix as soon as possible.

sorry for the inconvenience!

**Comment by Swaxer** [comment]
Much appreciated, I thought it would be in 4.10.0 as well since it was more or less already solved in #1001  :/ 
I can confirm that #1034 solves it 👍🏼🙏🏼

**Comment by xzoky** [comment]
Thanks for the quick fix @mohammedDehairy! I'm off for a few weeks so I'm not able to try it for myself, but I believe @Swaxer 😊

**Comment by mohammedDehairy** [comment]
will close this issue for now, please feel free to reopen or create a new issue if you're still experiencing issues!.

**Comment by xzoky** [comment]
Confirmed fixed in 4.10.1. Thanks @mohammedDehairy 🙂
